recursion.js

```diff
@@ -0,0 +1,110 @@
+// - ### Sum of all from 1 to n
+
+// Write a function that takes in a number as input and recursively finds the sum of all numbers up to and including that number.
+
+
+const sum = (n) => {
+    if(n <= 1) return n; 
+    return n + sum(n-1)
+    
+}
+console.log(sum(7))
+
+// ```js
+// input: 6
+// output: 21
+
+// //21 = 6 + 5 + 4 + 3 + 2 + 1
+// ```
+
+
+// - ### Multiply array
+
+// Write a function called `multArr` that takes in an array of numbers as an argument and recursively multiplies together all of the values in the array.
+
+const multArr = (arr) => {
+    if (arr.length === 0) return 1;
+    let el = arr.pop()
+    return multArr(arr) * el
+}
+
+// ```js
+console.log(multArr([2, 3, 5])); // returns 30
+console.log(multArr([5, 5, 1, 2])); //returns 50
+// ```
+
+// - ### Concatenate array
+
+// Write a function called `concatArr` that takes in an array of strings as an argument and recursively concatenates the strings together into a single string, with spaces between each original string.
+
+const concatArr = (arr) => {
+    if (arr.length === 0) return "";
+    let el = arr.pop()
+    return concatArr(arr) + el + " "
+}
+
+// ```js
+console.log(concatArr(['is', 'it', 'tomorrow'])); // returns 'is it tomorrow'
+console.log(concatArr(['or', 'just', 'the', 'end', 'of', 'time'])); //returns 'or just the end of time'
+// ```
+
+// - ### Sum evens
+
+// Write a function called `sumEvens` that takes in an array of numbers as an argument and recursively sums only the even numbers in the array.
+
+const sumEvens = (arr) => {
+    if (arr.length === 0) return 0;
+    let el = arr.pop()
+    if (el % 2 === 1) return 0;
```

Comment:
You are very close to having the correct answer. Changing line 58 would solve this problem. Think about what you want to return if the number is odd, as opposed to when the number is even.